common/changes/pcln-design-system/generic-banner-fix_2021-02-05-13-58.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "pcln-design-system",
+      "comment": "move where ctaText check is passed and add headerFontSize prop to GenericBanner",
+      "type": "minor"
```

Comment:
Sorry for being difficult but I think this can be a patch change now that no new props are added 😄

Reply:
SO DIFFICULT. but jk you're right. I will update